pkg/scheduler/framework/v1alpha1/interface.go

```diff
@@ -113,6 +113,17 @@ type PrebindPlugin interface {
 	Prebind(pc *PluginContext, p *v1.Pod, nodeName string) *Status
 }
 
+// UnreservePlugin is an interface for Unreserve plugins. This is an informational
+// extension point. If a pod was reserved and then rejected in a later phase, then
+// un-reserve plugins will be notified. Un-reserve plugins should clean up state
+// associated with the reserved Pod.
+type UnreservePlugin interface {
+	Plugin
+	// Unreserve is called by the scheduling framework when a reserved pod was
+	// rejected in a later phase.
+	Unreserve(pc *PluginContext, p *v1.Pod, nodeName string) *Status
```

Comment:
I don't think we need to have any return value for `Unreserve` plugins. These are called to inform plugins that an already reserved pod is not going to be bound. This extension point is only to inform plugins so that they can perform clean-up if needed.

Reply:
Agree!  Removed the return value.